gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxCardVerticalExample from './NxCardVerticalExample';
+import NxCardHorizontalExample from './NxCardHorizontalExample';
+import NxCardVerticalAltExample from './NxCardVerticalAlternateExample';
+
+const nxCardVerticalCode = require('!!raw-loader!./NxCardVerticalExample').default,
+    nxCardHorizontalCode = require('!!raw-loader!./NxCardHorizontalExample').default,
+    nxCardVerticalAltCode = require('!!raw-loader!./NxCardVerticalAlternateExample').default;
+
+const NxCardPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Cards can be displayed in two orientations, vertical and horizontal. The vertical and horizontal cards can be
+        transformed into one another with a simple class change on their container. There is also a vertical variant
+        supplied which displays the call out and description text side-by-side when the content allows, these cannot be
+        transformed into horizontal cards.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row nx-table-row--header">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">Top-Level</td>
+            <td className="nx-cell">
+              Basic card container class.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--row</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">
+              Container for vertically orientated cards (normal and variant). Normal vertical cards can be transformed
+              into horizontal cards by replacing this style with
+              {' '}<code className="nx-code">.nx-card-container--column</code> below.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--column</code></td>
```

Comment:
Which layout is the default?  Or is the intent that users _must_ specify one modifier or the other?

Reply:
> Which layout is the default? Or is the intent that users _must_ specify one modifier or the other?

I deliberately wrote it so the user must choose one layout or the other.